orca/request_opendap.py

```diff
@@ -0,0 +1,23 @@
+import re
+from xarray import open_dataset
+
+
+def request_opendap(url, out_file):
+    # note: interestingly enough we can't actually download the url we make but can access it using a dataset obj... hmmm
+    try:
+        d = open_dataset(url, engine="netcdf4")
+        d.to_netcdf(out_file)
+    except Exception as e:
+        raise e
```

Comment:
After this `try/except` block we should still return the file or do something tangible with it.